gnome-initial-setup/eos/hack-sound-client/eos-hack-sound-client.c

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (C) 2019 Endless Mobile. Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * Written by:
+ *     Cesar Fabian Orccon Chipana <cfoch.fabian@gmail.com>
+ */
+
+#include "eos-hack-sound-client.h"
```

Comment:
I took gis-driver.c as a reference, but I'd say everything was changed. Also not sure I because of the construct function I am using I should consider Xavier Claessens in the "Written by" comment.